lib/usd/utils/module.cpp

```diff
@@ -0,0 +1,8 @@
+#include "pxr/base/tf/pyModule.h"
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+TF_WRAP_MODULE
+{
+    TF_WRAP(MayaTransformAPI);
```

Comment:
For my own education, does this ensure that the MayaTransformAPI wrapper ends up in the mayaUsd.lib package?